library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -9,6 +9,7 @@ import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import android.view.View
+import androidx.test.InstrumentationRegistry
```

Comment:
It's deprecated, you can use androidx.test.platform.app.InstrumentationRegistry

Reply:
sure